Source/Project64/User Interface/Frame Per Second Class.cpp

```diff
@@ -58,7 +58,7 @@ void CFramePerSecond::Reset (bool ClearDisplay)
 
 void CFramePerSecond::UpdateViCounter ( void )
 {
-	if (m_iFrameRateType != FR_VIs && m_iFrameRateType != FR_PERCENT)
+	if (m_iFrameRateType != FR_VIs && m_iFrameRateType != FR_PERCENT && m_iFrameRateType != FR_VIs_DLs)

```

Comment:
I think this is needed to make sure the VI/s Counter updates.  if you press F4 the VI/s don't update at all without this line.